Horatio/Operations/AlertOperation.swift

```diff
@@ -65,18 +64,24 @@ class AlertOperation: Operation {
     }
 
     override func execute() {
-        guard let presentationContext = presentationContext else {
-            finish()
-
-            return
+        var presentationViewController: UIViewController?
```

Comment:
As above, this seems to be put in place to fix an app level bug, and does so inelegantly.